src/test/java/com/naveenautomation/tests/SpecialsPageTest.java

```diff
@@ -0,0 +1,43 @@
+package com.naveenautomation.tests;
+
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import com.naveenautomation.Base.TestBase;
+import com.naveenautomation.Pages.AccountLoginPage;
+import com.naveenautomation.Pages.MyAccountPage;
+import com.naveenautomation.Pages.SpecialsPage;
+
+public class SpecialsPageTest extends TestBase {
+
+	SoftAssert softAssert = new SoftAssert();
+	AccountLoginPage accountLoginPage;
+	MyAccountPage myAccountPage;
+
+	@BeforeMethod
+	public void setUp() {
+		launchBrowser();
+		accountLoginPage = new AccountLoginPage(driver, true).get();
+		myAccountPage = accountLoginPage.login("parul.verma421@gmail.com", "Parul4");
+
+	}
+
+	@Test
+	public void verifyUserIsAbleToAddItemFromSpecials() {
+		SpecialsPage specialPage = myAccountPage.clickOnSpeacialInExtras();
+		specialPage.clickOnAddToWishlist();
+
+		softAssert.assertEquals("Success: You have added Apple Cinema 30\" to your wish list!\n" + "×",
```

Comment:
use string methods and remove all unnecessary characters

Reply:
Code updated using String Method